src/NativeScript/ObjC/ObjCMethodCall.h

```diff
@@ -46,7 +46,8 @@ class ObjCMethodCall : public FFICall {
 
 private:
     ObjCMethodCall(JSC::VM& vm, JSC::Structure* structure)
-        : Base(vm, structure) {
+        : Base(vm, structure)
+        , _hasErrorOutParameter{ false } {
```

Comment:
[C++11: 5.3.4/15]: A new-expression that creates an object of type T initializes that object as follows: If the new-initializer is omitted, the object is default-initialized (8.5); if no initialization is performed, the object has indeterminate value.